diff --git a/dep/CommonLibSSE-NG/include/RE/R/ReferenceArray.h b/dep/CommonLibSSE-NG/include/RE/R/ReferenceArray.h
index df9bdcbe..fa7987ab 100644
--- a/dep/CommonLibSSE-NG/include/RE/R/ReferenceArray.h
+++ b/dep/CommonLibSSE-NG/include/RE/R/ReferenceArray.h
@@ -123,7 +123,7 @@ namespace RE
 				if (_wrapped) {
 					const auto SIZE = static_cast<std::uint32_t>(_unwrapped.size());
 					for (std::uint32_t i = 0; i < SIZE; ++i) {
-						(*_wrapped)[i].Pack(_unwrapped[i]);
+						(*_wrapped)[i].Pack(static_cast<T>(_unwrapped[i]));
 					}
 				}
 			}
diff --git a/dep/CommonLibSSE-NG/cmake/CommonLibSSE.cmake b/dep/CommonLibSSE-NG/cmake/CommonLibSSE.cmake
index c8e46627..cb608971 100644
--- a/dep/CommonLibSSE-NG/cmake/CommonLibSSE.cmake
+++ b/dep/CommonLibSSE-NG/cmake/CommonLibSSE.cmake
@@ -106,8 +106,10 @@ function(target_commonlibsse_properties TARGET)
     endif ()
 
     file(WRITE "${commonlibsse_plugin_file}"
-            "#include \"REL/Relocation.h\"\n"
             "#include \"SKSE/SKSE.h\"\n"
+            "#include \"REL/Relocation.h\"\n"
+            "\n"
+            "using namespace std::literals;"
             "\n"
             "SKSEPluginInfo(\n"
             "    .Version = ${commonlibsse_plugin_version},\n"

